### DISCUSSION TITLE
How to make VCPKG run with older CMake Version?

### DISCUSSION THREAD

**Discussion by zhang-stephen** [microsoft/vcpkg#38628] [Q&A]
the master branch of VCPKG has updated minimum required CMake version 3.29.x, but there is CMake 3.28.x available in my workspace, and I cannot update CMake(no permissions to do that). 

How to ignore the CMake version required, except downgrading VCPKG?

**Comment by autoantwort** [comment]
You can manually change the [scripts/vcpkgTools.xml](https://github.com/microsoft/vcpkg/blob/master/scripts/vcpkgTools.xml) file

**Reply by zhang-stephen** [reply]
well, this is not a proper way in my opinion. I want to use VCPKG as a git submodule, it cannot work if files from VCPKG were modified.

**Reply by autoantwort** [reply]
Afaik there is no other way
